packages/hop-node/src/config/fileOps.ts

```diff
@@ -22,12 +25,12 @@ import { recoverKeystore } from 'src/keystore'
 const logger = new Logger('config')
 
 export const defaultEnabledWatchers: { [key: string]: boolean } = {
```

Comment:
The concept of `defaultEnabledWatchers` has been removed because all watchers are disabled by default. Resolved with https://github.com/hop-protocol/hop/pull/346/commits/36a93478a3aa1d9fccd0c560ea29b4a32c1059bb